.gitlab/benchmark.yml

```diff
@@ -110,14 +109,13 @@ build-adp-check-comparison-image:
     - aws ecr get-login-password --region us-west-2 --profile ${AWS_NAMED_PROFILE} | docker login --username AWS --password-stdin ${SMP_ECR_HOST}
     - docker buildx build
       --platform linux/amd64
-      --file ./docker/Dockerfile.agent-data-plane
+      --file ./docker/Dockerfile.adp-checks-smp
```

Comment:
And then you can reference it here:
```
--file *adp-checks-smp
```
That way if it ever changes you only update it in one place.